### DISCUSSION TITLE
Import historical price data?

### DISCUSSION THREAD

**Discussion by hvddrift** [ghostfolio/ghostfolio#2500] [Q&A] [answered]
I used to use Adminer to manually add entries for historical prices of some of my investments, but I stopped because it was too cumbersome. Is it possible to use the import function to (for example) add the last five years of daily close prices?

**Comment by dtslvr** [comment, marked as answer]
Hi @hvddrift

With the latest version ([Ghostfolio 2.11.0](https://github.com/ghostfolio/ghostfolio/releases/tag/2.11.0)), we have added support to import historical market data in `csv` format. You can find the feature in _Market Data_ of the _Admin Control_ panel. At the moment there is no API available.

We are happy to get your feedback.

PR for reference: https://github.com/ghostfolio/ghostfolio/issues/2046

**Reply by hvddrift** [reply]
Thank you Thomas. a small bit of excel manipulation and I have 4000 entries of historical data. Fantastic.

I am blaming my age, but I have a few Vanguard Managed Funds that I track, they are Manual (no online updates) 

But, do you think I can remember  how I added these to my portfolio... Was it when I spun up "adminer" perhaps? Before I go down this rabbit hole again, are there ways to add manual entries into yet that I cant see?

**Reply by dtslvr** [reply]
Yes, with the latest version ([Ghostfolio 2.12.0](https://github.com/ghostfolio/ghostfolio/releases/tag/2.12.0)) you can create asset profiles with `MANUAL` data source from _Market Data_ of the _Admin Control_ panel.
